winpr/libwinpr/utils/image.c

```diff
@@ -192,7 +192,7 @@ int winpr_image_bitmap_read_fp(wImage* image, FILE* fp)
 
 	image->bitsPerPixel = bi.biBitCount;
 	image->bytesPerPixel = (image->bitsPerPixel / 8);
-	image->scanline = (bi.biSizeImage / bi.biHeight);
```

Comment:
I think the same problem (negative scanline if biHeight < 0) also exists in winpr_image_bitmap_read_buffer